File: calculate_points.rb
require 'points_calculator'

lines = File.open('sbrr-2024-points.csv') { |f| f.readlines }
standings = []
lines.each do |line|
  standings << PointsCalculator.new(line)
end

standings.sort { |a, b| b.total_with_drops <=> a.total_with_drops }.each do |driver|
  puts driver.display
end

File: calculate_points.py
import csv
from points_calculator import PointsCalculator


points = []
with open("sbrr-2024-points.csv", newline="") as file:
    rows = csv.reader(file)
    for row in rows:
        points.append(PointsCalculator(row))

points.sort(key=lambda p: p.total_with_drops(), reverse=True)
for p in points:
    print(p.display())
